### DISCUSSION TITLE
Mail envelopes?

### DISCUSSION THREAD

**Discussion by strarsis** [ciur/papermerge#460] [Q&A]
Sometimes it makes sense to also archive the envelope of a mail document.
Does Papermerge require special treatment for envelopes?
Can those envelopes be linked to the mail document?

**Comment by ciur** [comment]
What exactly do you mean by mail envelope ? is there a special file type ?

**Reply by strarsis** [reply]
I mean can it associate the mail envelope with the letter? Can it understand the address on it? Also envelopes come in specific physical formats.

**Reply by ciur** [reply]
Now I understand what you mean.
No, there is no such feature to link (associate) two documents.
However, there are at least two workarounds that see, which may accomplish what you want:
1. Create one folder for each physical letter and place both envelop  and the letter in that folder
2. Or, in case you don't like folder idea, you may upload both documents (envelop and the content of the letter) as files with same title (uploaded documents may have same title) and add tag e.g. 'content' to one and tag 'envelop' to the other document

Basically option 1. is to use folders (and you will have one folder for each physical letter)
and option 2 is to use tags and name different scans of the same "physical letter" with same title.
